### PR TITLE
pacparser: fix compilation with clang

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/pacparser/package.py
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]: I think this is the Spack way of explicitly setting compiler. @tldahlgren correct me if I'm wroing.

```suggestion
        make('CC="%s"' % self.compiler.cc, 'CXX="%s"' % self.compiler.cxx, '-C', 'src')
```

File: var/spack/repos/builtin/packages/pacparser/package.py
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]:
```suggestion
            make('CC="%s"' % self.compiler.cc, 'CXX="%s"' % self.compiler.cxx, '-C', 'src', 'pymod')
```

File: var/spack/repos/builtin/packages/pacparser/package.py
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]: I did not see any issues building pacparser in parallel, but I didn't yet test it with clang.

```suggestion
    @property
    def parallel(self):
        return self.spec.satisfies('@1.3.7')
```

File: var/spack/repos/builtin/packages/pacparser/package.py
Comment (iarspider) [inline var/spack/repos/builtin/packages/pacparser/package.py]: Please update your fork from upstream.

**Review by iarspider — CHANGES_REQUESTED** [review]

**Review by iarspider — DISMISSED** [review]
LGTM. Did not test with clang myself.

**Comment by iarspider** [comment]
@alalazo  @tldahlgren I don't have write access, can't merge it myself.

**Comment by iarspider** [comment]
@vvolkl 
> var/spack/repos/builtin/packages/pacparser/package.py:34: [E501] line too long (100 > 88 characters)

**Review by iarspider — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**
